.gitignore

```diff
@@ -1,3 +1,6 @@
+# Mac
+*.DS_Store
```

Comment:
While it's of course not a problem to include this here, this is more a case for your global, user-wide gitignore, not individual project gitignore:

```
git config core.excludesfile ~/.gitignore
echo ".DS_Store" >> ~/.gitignore
echo "__MACOSX" >> ~/.gitignore
```